python_ta/reporters/templates/stylesheet.css

```diff
@@ -29,6 +29,21 @@ header {
     overscroll-behavior-y: contain;
 }
 
+.sidebar ul {
```

Comment:
Add to this section to reduce the amount of whitespace before each list element.